File: strategy/marvin/test/move/defend/mrltestcorner.lua
local MrlTestCorner = Class("Test.Move.Defend.MrlTestCorner", require "group/move/mrltestcorner")

local G = (require "../base/world").Geometry
MrlTestCorner.DEBUG_GOOD_POS = {
		{Vector(G.FieldWidthHalf / 2, 4 * G.FieldHeightHalf / 5), Vector(G.FieldWidthHalf, G.FieldHeightHalf)},
		{Vector(-G.FieldWidthHalf / 2, 4 * G.FieldHeightHalf / 5), Vector(-G.FieldWidthHalf, G.FieldHeightHalf)},
--		{Vector(), Vector()}
}

return MrlTestCorner

File: strategy/typescript/glados/test/move/defend/mrltestcorner.ts
let MrlTestCorner = Class("Test.Move.Defend.MrlTestCorner", require "group/move/mrltestcorner")

let G = (require "+/base/world").Geometry
MrlTestCorner.TEST_BALL_START_RECTS = {
		{new Vector(G.FieldWidthHalf / 2, 4 * G.FieldHeightHalf / 5), new Vector(G.FieldWidthHalf, G.FieldHeightHalf)},
		{new Vector(-G.FieldWidthHalf / 2, 4 * G.FieldHeightHalf / 5), new Vector(-G.FieldWidthHalf, G.FieldHeightHalf)},
//		{new Vector(), new Vector()}
}

return MrlTestCorner
